<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminmanageitems</class>
 <widget class="QDialog" name="adminmanageitems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>97</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_name">
      <property name="text">
       <string>Update Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_price">
      <property name="text">
       <string>Update price demand:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_description">
      <property name="text">
       <string>Update description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_availability">
      <property name="text">
       <string>Update availability:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_stock">
      <property name="text">
       <string>update stock:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_period">
      <property name="text">
       <string>Update rental period: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>131</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_price">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_description"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_availability"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_stock"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_period"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>76</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Item name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_price">
      <property name="text">
       <string>Item price:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>Item description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_availability">
      <property name="text">
       <string>Item availbility:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_stock">
      <property name="text">
       <string>Item stock:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_period">
      <property name="text">
       <string>item rental period:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_index">
      <property name="text">
       <string>vector index (i)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QPushButton" name="pushButton_add">
      <property name="text">
       <string>add to vector</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QPushButton" name="pushButton_left">
      <property name="text">
       <string>goleft</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QPushButton" name="pushButton_right">
      <property name="text">
       <string>goright</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_delete">
      <property name="text">
       <string>delete from vector</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
